recipes/consul.rb

```diff
@@ -1,8 +1,6 @@
 include_recipe 'optoro_consul::client'
 
 optoro_consul_service 'mysql' do
-  checks [
-    { script: 'service mysql status', interval: '5s' }
-  ]
   port 3306
+  params node['optoro_consul']['service']
```

Comment:
@zadunn Yes!

It allows more flexibility! We can supply arbitrary attributes without having to change the API